components/profile/ImportedPublicationsSection.js

```diff
@@ -78,18 +79,33 @@ const ImportedPublicationsSection = ({
     )
   }, [pageNumber, publications])
 
-  if (!publicationsToDisplay.length) return null
-  return (
-    <div>
-      <NoteList notes={publicationsToDisplay} displayOptions={displayOptions} />
-      <PaginationLinks
-        currentPage={pageNumber}
-        itemsPerPage={pageSize}
-        totalCount={totalCount}
-        setCurrentPage={setPageNumber}
-        options={{ noScroll: true }}
+  if (!publicationsToDisplay.length) {
+    return (
+      <ProfileSection
+        title="Imported Publications"
+        instructions="Did not find publications listing you as an author in DBLP and other sources."
       />
-    </div>
+    )
+  }
+
+  return (
+    <ProfileSection
+      title="Imported Publications"
+      instructions="Below is a list of publications imported from DBLP and other sources that
+        include you as an author. To remove any publications you are not actually an author of
+        from your profile, click the minus sign next to the title."
+    >
+      <div>
+        <NoteList notes={publicationsToDisplay} displayOptions={displayOptions} />
+        <PaginationLinks
+          currentPage={pageNumber}
+          itemsPerPage={pageSize}
+          totalCount={totalCount}
+          setCurrentPage={setPageNumber}
+          options={{ noScroll: true }}
+        />
+      </div>
+    </ProfileSection>
```

Comment:
I think what you are saying for someone without publications would result in a section that says "Imported Publications" and nothing more.  It would also be easy to take out the "Imported Publications" section entirely. 

I think it adding the "Did not find..." added clarity.

Do you want me to remove the extra instruction?